fact-bounty-flask/app.py

```diff
@@ -38,20 +38,25 @@ def test(coverage):
         sys.exit(subprocess.call(sys.argv))
     
     import unittest
-    tests = unittest.TestLoader().discover('tests')
-    unittest.TextTestRunner(verbosity=2).run(tests)
-    if COV:
```

Comment:
Include COV, it will help for test coverage

Reply:
Test coverage was not working when I executed the test.
Let me figure that out!